x-pack/filebeat/module/microsoft/defender_atp/config/atp.yml

```diff
@@ -29,6 +29,7 @@ request.transforms:
 
 response.split:
   target: body.value
+  ignore_empty_value: true
```

Comment:
Do we have any test cases in our logs that have an empty `value` field?

Reply:
We do in the integration, but not here.

Reply:
I've come up with an approach that serves to test this. It's not pretty, but it work.